src/app/modules/item/pages/item-display/item-display.component.html

```diff
@@ -33,7 +33,7 @@
         <p-button type="button" (click)="skipSave.emit()" i18n-label label="Skip"></p-button>
       </p>
     </div>
-    <iframe *ngIf="iframeSrc$ | async as iframeSrc" [style.height]="iframeHeight$ | async" #iframe [src]="iframeSrc"></iframe>
+    <iframe *ngIf="iframeSrc$ | async as iframeSrc" #iframe [src]="iframeSrc" algFullHeightContent></iframe>
```

Comment:
It simplifies very well the height handling :+1:
However I'm not sure that's _exactly_ what we want here: some tasks have a full height because their configuration specifies `autoHeight: true`, but for others with `autoHeight: undefined` or `autoHeight: false`, we need to apply the task-requested height on the iframe.
At least it was how it was done until now, I have no idea if that behavior is supposed to change. @smadbe ?

Reply:
Yes, Thomas is right, there are several "types of height-management" for tasks, some have an iframe and handle the scrolling themselves (such as the codecast demo (6314563572459051248)), some have no scrolling in the iframe but use the "app scrolling" (such as "Enseignants: description de la séquence" id 837046206729127555). 
It was working as expected (except the margin) before, this should not be broken.

Reply:
Fixed